### ISSUE TITLE
Support standalone batchnorm in XNNPACK backend

### ISSUE THREAD

**Issue by GregoryComer** [pytorch/executorch#16627]
Support un-fused batch norm in XNNPACK.

cc @digantdesai @cbilgin

**Comment by GregoryComer** [comment]
Draft is up in https://github.com/pytorch/executorch/pull/16533.
